tests/test_topline_dashboard.py

```diff
@@ -0,0 +1,225 @@
+import json
+
+from click.testing import CliRunner
+
+import boto3
+import pytest
+from moto import mock_s3
+from mozetl.topline import topline_dashboard as topline
+from mozetl.topline.schema import historical_schema, topline_schema
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture(scope="session")
```

Comment:
I'll address this in another PR, since it'll be touching a couple of different files.